diff --git a/node_modules/@stripe/stripe-react-native/ios/NewArch/ConnectAccountOnboardingViewComponentView.mm b/node_modules/@stripe/stripe-react-native/ios/NewArch/ConnectAccountOnboardingViewComponentView.mm
new file mode 100644
index 0000000..0e0751d
--- /dev/null
+++ b/node_modules/@stripe/stripe-react-native/ios/NewArch/ConnectAccountOnboardingViewComponentView.mm
@@ -0,0 +1,9 @@
+#import <React/RCTFabricComponentsPlugins.h>
+#import <React/RCTViewComponentView.h>
+
+// On older versions of react-native every single spec is linked so we need
+// to include a dummy implementation to avoid linker errors.
+Class<RCTComponentViewProtocol> ConnectAccountOnboardingViewCls(void)
+{
+  return RCTViewComponentView.class;
+}
diff --git a/node_modules/@stripe/stripe-react-native/ios/NewArch/PaymentMethodMessagingElementViewComponentView.mm b/node_modules/@stripe/stripe-react-native/ios/NewArch/PaymentMethodMessagingElementViewComponentView.mm
new file mode 100644
index 0000000..9f6758c
--- /dev/null
+++ b/node_modules/@stripe/stripe-react-native/ios/NewArch/PaymentMethodMessagingElementViewComponentView.mm
@@ -0,0 +1,9 @@
+#import <React/RCTFabricComponentsPlugins.h>
+#import <React/RCTViewComponentView.h>
+
+// On older versions of react-native every single spec is linked so we need
+// to include a dummy implementation to avoid linker errors.
+Class<RCTComponentViewProtocol> PaymentMethodMessagingElementViewCls(void)
+{
+  return RCTViewComponentView.class;
+}
