diff --git a/node_modules/lottie-react-native/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt b/node_modules/lottie-react-native/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt
index e4bfb4a..7e28cdf 100644
--- a/node_modules/lottie-react-native/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt
+++ b/node_modules/lottie-react-native/android/src/main/java/com/airbnb/android/react/lottie/LottieAnimationViewPropertyManager.kt
@@ -103,9 +103,9 @@ class LottieAnimationViewPropertyManager(view: LottieAnimationView) {
             if (it.size() > 0) {
                 val textDelegate = TextDelegate(view)
                 for (i in 0 until textFilters!!.size()) {
-                    val current = textFilters!!.getMap(i)
-                    val searchText = current.getString("find")
-                    val replacementText = current.getString("replace")
+                    val current = textFilters!!.getMap(i) ?: continue
+                    val searchText = current!!.getString("find")
+                    val replacementText = current!!.getString("replace")
                     textDelegate.setText(searchText, replacementText)
                 }
                 view.setTextDelegate(textDelegate)
@@ -213,8 +213,8 @@ class LottieAnimationViewPropertyManager(view: LottieAnimationView) {
         colorFilters?.let { colorFilters ->
             if (colorFilters.size() > 0) {
                 for (i in 0 until colorFilters.size()) {
-                    val current = colorFilters.getMap(i)
-                    parseColorFilter(current, view)
+                    val current = colorFilters.getMap(i) ?: continue
+                    parseColorFilter(current!!, view)
                 }
             }
         }
diff --git a/node_modules/lottie-react-native/android/src/newarch/com/airbnb/android/react/lottie/LottieAnimationViewManager.kt b/node_modules/lottie-react-native/android/src/newarch/com/airbnb/android/react/lottie/LottieAnimationViewManager.kt
index af9771e..ddb6c1a 100644
--- a/node_modules/lottie-react-native/android/src/newarch/com/airbnb/android/react/lottie/LottieAnimationViewManager.kt
+++ b/node_modules/lottie-react-native/android/src/newarch/com/airbnb/android/react/lottie/LottieAnimationViewManager.kt
@@ -37,7 +37,7 @@ class LottieAnimationViewManager :
     private val delegate: ViewManagerDelegate<LottieAnimationView>
 
     init {
-        delegate = LottieAnimationViewManagerDelegate(this)
+        delegate = LottieAnimationViewManagerDelegate<LottieAnimationView, LottieAnimationViewManager>(this)
     }
 
     private fun getOrCreatePropertyManager(view: LottieAnimationView): LottieAnimationViewPropertyManager {
