diff --git a/node_modules/react-native-maps/ios/AirMaps/AIRMap.m b/node_modules/react-native-maps/ios/AirMaps/AIRMap.m
index dbc85fb..ffb2a36 100644
--- a/node_modules/react-native-maps/ios/AirMaps/AIRMap.m
+++ b/node_modules/react-native-maps/ios/AirMaps/AIRMap.m
@@ -133,7 +133,8 @@ - (void)insertReactSubview:(id<RCTComponent>)subview atIndex:(NSInteger)atIndex
           [self insertReactSubview:(UIView *)childSubviews[i] atIndex:atIndex];
         }
     }
-    [_reactSubviews insertObject:(UIView *)subview atIndex:(NSUInteger) atIndex];
+    NSUInteger safeIndex = MIN((NSUInteger)MAX(atIndex, 0), _reactSubviews.count);
+    [_reactSubviews insertObject:(UIView *)subview atIndex:safeIndex];
 }
 #pragma clang diagnostic pop
 
