diff --git a/node_modules/react-native/Libraries/promiseRejectionTrackingOptions.js b/node_modules/react-native/Libraries/promiseRejectionTrackingOptions.js
index 5a12d2d..947958d 100644
--- a/node_modules/react-native/Libraries/promiseRejectionTrackingOptions.js
+++ b/node_modules/react-native/Libraries/promiseRejectionTrackingOptions.js
@@ -36,8 +36,12 @@ let rejectionTrackingOptions: $NonMaybeType<Parameters<enable>[0]> = {
       }
       // It could although this object is not a standard error, it still has stack information to unwind
       // $FlowFixMe ignore types just check if stack is there
-      if (rejection.stack && typeof rejection.stack === 'string') {
-        stack = rejection.stack;
+      try {
+        if (rejection.stack && typeof rejection.stack === 'string') {
+          stack = rejection.stack;
+        }
+      } catch (_e) {
+        // Error.stack getter may throw with an invalid receiver on Hermes
       }
     }

